arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -2018,6 +1882,32 @@
       },
       "x-ms-azure-resource": true
     },
+    "GeoFilter": {
+      "description": "Geo filter of a CDN endpoint.",
+      "type": "object",
+      "required": [
+        "relativePath",
+        "action",
+        "countryCodes"
+      ],
+      "properties": {
+        "relativePath": {
+          "description": "Relative path apply geo filter.",
```

Comment:
The English phrase of the description is in correct. "Relative path applicable to geo filter."
What can the user provide over over here? Can you provide an example ?